### PR TITLE
Updated the file name 

### PR DESCRIPTION
**Pull request by sushma-4**
**Description of the Pull Request (PR):**

Write your description of the PR here. Be sure to include as much background,
and details necessary for the reviewers to understand exactly what this is
fixing or enhancing.

**This fixes or addresses the following GitHub issues:**

- Fixes #

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: docs/remote.go [hunk 1]
```diff
@@ -57,8 +57,8 @@ const (
 	RemoteListUse   string = `list`
 	RemoteListShort string = `List all singularity remote endpoints that are configured`
 	RemoteListLong  string = `
-	The 'remote list' command lists all remote endpoints configured for use. If you 
-	have set a remote as a default, its name will be encompassed by brackets.`
+	The 'remote list' command lists all remote endpoints configured for use. If a remote
+	has been set as default, its name will be encompassed by brackets.`
```
Comment (WestleyK) [inline docs/remote.go]: Maybe?

```suggestion
	is in use, its name will be encompassed by brackets.`
```

CC @ikaneshiro

**Review by WestleyK — COMMENTED** [review]
One suggestion... Otherwise looks good.

File: docs/remote.go (hunk 1 above)
Comment (ikaneshiro) [reply docs/remote.go]: @WestleyK's suggestion feels like the best way to phrase it

**Review by WestleyK — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**